### PR TITLE
Fix mkcloud ha case

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by dirkmueller** [comment]
(completely untested, just debugging something for QAM)

File: scripts/mkcloud [hunk 1]
```diff
@@ -1020,7 +1020,7 @@ Optional
         Examples:
             3 clusters: clusterconfig='data=2:network=3:services=5'
             2 clusters: clusterconfig='services+data=2:network=3'
-            1 cluster:  clusterconfig='data+network+services=2'
+            1 cluster:  clusterconfig='services+data+network=2'
```
Comment (aspiers) [inline scripts/mkcloud]: As discussed on IRC I think this commit should be dropped.

Comment (aspiers) [inline scripts/mkcloud]: Or at least, the commit message justifying it should be changed.  I'm OK with it suggesting that the cluster is named `services`, but it shouldn't still work if it's called `data` or even `network`.

**Comment by bmwiedemann** [comment]
+0.5 - triggered a HA testrun to give it some test coverage. @jdsn should have a look at the change.

**Comment by aspiers** [comment]
Gate failed with surprising `neutron::server` recipe issue:

```
10:01:28 ================================================================================
10:01:28 Recipe Compile Error in /var/chef/cache/cookbooks/neutron/recipes/server.rb
10:01:28 ================================================================================
10:01:28 
10:01:28 NoMethodError
10:01:28 -------------
10:01:28 undefined method `[]' for nil:NilClass
10:01:28 
10:01:28 Cookbook Trace:
10:01:28 ---------------
10:01:28   /var/chef/cache/cookbooks/neutron/recipes/server.rb:133:in `from_file'
10:01:28 
10:01:28 Relevant File Content:
10:01:28 ----------------------
10:01:28 /var/chef/cache/cookbooks/neutron/recipes/server.rb:
10:01:28 
10:01:28 126:    owner node[:neutron][:user]
10:01:28 127:    group node[:neutron][:group]
10:01:28 128:    mode 0755
10:01:28 129:    action :create
10:01:28 130:    only_if { node[:platform_family] == "debian" }
10:01:28 131:  end
10:01:28 132:  
10:01:28 133>> vlan_start = node[:network][:networks][:nova_fixed][:vlan]
10:01:28 134:  num_vlans = node[:neutron][:num_vlans]
10:01:28 135:  vlan_end = [vlan_start + num_vlans - 1, 4094].min
10:01:28 136:  
10:01:28 137:  gre_start = [node[:neutron][:gre][:tunnel_id_start], 1].max
10:01:28 138:  gre_end = [node[:neutron][:gre][:tunnel_id_end], 2147483647].min
10:01:28 139:  
10:01:28 140:  vni_start = [node[:neutron][:vxlan][:vni_start], 0].max
10:01:28 141:  vni_end = [node[:neutron][:vxlan][:vni_end], 16777215].min
```

**Comment by aspiers** [comment]
Filed as https://trello.com/c/LQNYjA3i

**Comment by jdsn** [comment]
:-1: I will propose an alternative, as this change changes the assumptions behind the variables (maybe the naming needs to be improved as well)

**Comment by jdsn** [comment]
:+1: right, the name of the services cluster may be different, so using this variable is the correct way

**Pull request merged**